sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc

```diff
@@ -440,10 +456,30 @@ architecture.
 |-
 |AMD GCN GFX9 (Vega) architecture.
 
+|`amd_gpu_gfx909`
+|-
+|AMD GCN GFX9 (Vega) architecture.
+
 |`amd_gpu_gfx90a`
 |-
 |AMD GCN GFX9 (Vega) architecture.
 
+|`amd_gpu_gfx90c`
+|-
+|AMD GCN GFX9 (Vega) architecture.
+
+|`amd_gpu_gfx940`
+|-
+|AMD GCN GFX9 (Vega) architecture.
+
+|`amd_gpu_gfx941`
+|-
+|AMD GCN GFX9 (Vega) architecture.
+
+|`amd_gpu_gfx942`
+|-
+|AMD GCN GFX9 (Vega) architecture.
+
```

Comment:
The description of all these new arch entries is the same.  Is that really right?  What's the difference between them?

Reply:
No. Vega and CDNA. Thanks.

Reply:
I don't really know these architectures, but many of the new entries have exactly the same description.  For example, `amd_gpu_gfx940`, `amd_gpu_gfx941`, and `amd_gpu_gfx942` are all described as "AMD GCN GFX9 (CDNA 3) architecture".

Is there really no difference between these three?  If that is the case, why do we have three enumerators?

Reply:
Is it okay to group them ?
```
|`amd_gpu_gfx940`
|`amd_gpu_gfx941`
|`amd_gpu_gfx942`
|-
|AMD GCN GFX9 (CDNA 3) architecture.
```

Reply:
This doesn't address my question.  If they are all exactly the same, why do we have three enumerators?  If they are different in some way, the description should explain the difference.

Reply:
I updated the doc. If it still doesn't look good, please have a AMD product manager review the correctness of the names of the architectures. Thanks.

Reply:
Thanks, @jinz2014.